recipes-devtools/triton/triton-tensorrt-backend/0002-add-support-for-kUINT8.patch

```diff
@@ -6,6 +6,8 @@ Subject: [PATCH 2/2] add support for kUINT8
 Based on the following commit:
 https://github.com/triton-inference-server/tensorrt_backend/commit/114d0cd241eecb2ff78bb5ad4cdb4c138b2ef845
 
+Upstream-Status:  Backport [https://github.com/triton-inference-server/tensorrt_backend/commit/114d0cd241eecb2ff78bb5ad4cdb4c138b2ef845
```

Comment:
One too many blanks between 'Status:' and 'Backport' here. The check is quite unforgiving about such trivial formatting matters, unfortunately.